src/test/java/dk/kb/labsapi/ImageExportTest.java

```diff
@@ -0,0 +1,151 @@
+package dk.kb.labsapi;
+
+import dk.kb.labsapi.config.ServiceConfig;
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.apache.solr.common.SolrDocumentList;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.awt.*;
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * IMPORTANT: All this only works with a proper setup and contact to Solr
```

Comment:
Yeah, quite a problem here. But it's not yours. We need to find a more general solution to this problem.

The unit tests themselves seems very illustrative and code-covering.